ice/recipes/eval_synthesize.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+
+# from ice.recipes.synthesize_ft import synthesize_ft_from_df
```

Comment:
It's clumsy that you have to import the recipe, then specify it in code, in order to eval it

But I think it's probably actually fine for this case of "run a special eval"